packages/integrations/node/src/index.ts

```diff
@@ -69,12 +73,6 @@ export default function createIntegration(userOptions: UserOptions): AstroIntegr
 					assets: config.build.assets,
 				};
 				setAdapter(getAdapter(_options));
-
-				if (config.output === 'static') {
-					logger.warn(
-						`\`output: "server"\` or  \`output: "hybrid"\` is required to use this adapter.`,
-					);
-				}
```

Comment:
This adapter force a server build, so this error message cannot be triggered anymore.